libloragw/src/loragw_gps.c

```diff
@@ -640,6 +900,10 @@ int lgw_gps_get(struct timespec *utc, struct timespec *gps_time, struct coord_s
         /* Fractional part in nanoseconds */
         gps_time->tv_nsec = (long)(fractpart * 1E9);
     }
+    if (gps_time_acc != NULL) {
+        gps_time_acc->tv_sec = 0;
+        gps_time_acc->tv_nsec = gps_tAcc_ns;
```

Comment:
Why?

Reply:
Sorry, that's some older cruft that I need to back out. This PR is still WIP. But, that was my original plant to report time accuracy.